sunflower-land/sunflower-land
# [FEAT] Add Season Loot to Chests

# Description

- Adds `Grape Pants` and `Paper Reed` to the in game chests. These are seasonal items.
- Adds `Grape Pants` boost

## How to Test

1. Add Grape Pants to wardrobe
2. Equip Grape Pants
3. Plant Grape in Greenhouse and check boost is applied

## Screenshots

<img width="265" alt="1" src="https://github.com/user-attachments/assets/cf3e361f-91e3-4bb5-8549-468de9ef051b">
<img width="567" alt="2" src="https://github.com/user-attachments/assets/885b0ab3-f66b-40b0-99f0-ad1c4dbde429">


## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

localhost

# Checklist:
